lib/refile/s3.rb

```diff
@@ -52,7 +46,7 @@ def initialize(access_key_id:, secret_access_key:, region:, bucket:, max_size: n
     verify_uploadable def upload(uploadable)
       id = @hasher.hash(uploadable)
 
-      if uploadable.is_a?(Refile::File) and uploadable.backend.is_a?(S3) and uploadable.backend.access_key_id == access_key_id
```

Comment:
Can we just strike this? I think copying between two different S3 accounts will not work. Is there some other way to determine which account we're working with?